api/models/report.py

```diff
@@ -4,14 +4,15 @@
 import datetime
 
 class Report(db.Document):
-    salary = db.IntField(required=True)
     zipcode = db.IntField(required=True)
     credit_score = db.IntField(required=True)
+    salary = db.IntField(required=True)
     monthly_debt = db.IntField(required=True)
     downpayment_savings = db.IntField(required=True)
+    mortgage_term = db.IntField(default=0)
     downpayment_percentage = db.IntField(required=True)
-    rent = db.IntField(default=0)
     goal_principal = db.IntField(default=0)
+    rent = db.IntField(default=0)
```

Comment:
Cool, glad to see we have some defaults so we can let the use know they aren't required on the frontend!

Reply:
Absolutely!